clients/gtest/trsm_gtest.yaml

```diff
@@ -20,6 +20,14 @@ Definitions:
 
   - &alpha_range [ 1.0, -5.0 ]
 
+  - &complex_alpha_range
```

Comment:
I don't think you need separate test cases for real/complex precision. You can just use `*single_double_precisions_complex_real` as in `trtri_gtest.yaml` and have a single alpha range (like this one), `alphai` should just be ignored in the real cases.

Reply:
Thanks for the suggestion. I was following gemm style but this is cleaner.